pkg/cmd/factory/remote_resolver_test.go

```diff
@@ -37,8 +37,41 @@ func Test_remoteResolver(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, 2, len(remotes))
 
-	assert.Equal(t, "upstream", remotes[0].Name)
-	assert.Equal(t, "fork", remotes[1].Name)
+	assert.Equal(t, "fork", remotes[0].Name)
+	assert.Equal(t, "upstream", remotes[1].Name)
```

Comment:
I would expect this to be `origin` not `upstream`. Is there something else affecting the sort order?